src/aliases/mapper/Befehle/minit.lua

```diff
@@ -1,16 +1,41 @@
-
 -- Dies setzt die Karte auf Werkszustand zurueck.
 
-for i,name in pairs(getRooms()) do
-    deleteRoom(i)
+-- Sonderfälle, behandeln wir irgendwann in der Zukunft
+if gmcp.MG.room.info == nil or gmcp.MG.room.info.id == nil then
+    echoM("Bitte begebe dich in einen anderen Raum und versuche es noch einmal! Habe derzeit keinerlei Informationen ueber deinen aktuellen Standort im MorgenGrauen :(")
+    return
+end
+
+-- vorhandene Räume löschen
+for id, _ in pairs(getRooms()) do
+    deleteRoom(id)
+end
+
+-- vorhandene Gebiete löschen, ausser "Default Area"
+for _, id in pairs(getAreaTable()) do
+    if id > -1 then
+        deleteArea(id)
+    end
+end
+
+-- MapUserData löschen
+clearMapUserData()
+
+-- ersten Raum aus aktuellen GMCP Daten erstellen
+local hash = gmcp.MG.room.info.id
+
+mapper.currentHash = hash
+
+local newRoom = createRoom(mapper.currentArea, hash)
+local roomName = gmcp.MG.room.info.short
+setRoomName(newRoom, roomName)
+
+-- im neuen Raum alle sichtbaren Ausgänge prüfen und ggf. Stubs erzeugen
+for _, exitname in pairs(gmcp.MG.room.info.exits) do
+    addStubExit(newRoom, exitname)
 end
 
-addRoom(1)
-setRoomArea(1, findArea("world"))
```

Comment:
Die Idee wird nicht mehr aufgegriffen? In welcher Area wird der Spieler nach minit mappen?

edit: Tests zeigen, die aktuelle Area wird nicht gelöscht, vermutlich streikt deleteArea da?
Jedenfalls mappt man in der Area weiter, in der man minit ausführt. Könnte soweit passen.

Reply:
Dafür ist jetzt
 ```lua
local newRoom = createRoom(mapper.currentArea, hash)
```
zuständig.

mapper.currentArea ist da bereits "world" (scripts/mapper/Initialisierung.lua).

Reply:
... bzw. ist mapper.currentArea die letzte vom Spieler gesetzte Area. Stimmt. Brr. Nicht dran gedacht. Am besten bei minit auch die currentArea auf "world" setzen? Sonst haben wir wieder einen potentiellen "Restposten" übrig.

Reply:
Ja, zu dem Ergebnis bin ich auch gekommen.

Reply:
Notiz an mich, damit ich es nicht über Nacht vergesse:

- prüfen ob currentHash dazu führt das nach minit manchmal das Mappen nicht mehr funktioniert, da handleRoomInfo testet ob currentHash und gmcp.MG.room.info.id den selben Wert haben was dann als kein-neuer-Raum-betreten zählt